install/kubernetes/helm/istio/test-values/values-e2e.yaml

```diff
@@ -16,7 +16,7 @@ global:
     mode: REGISTRY_ONLY
 
 prometheus:
-  scrapeInterval: 5s
+  scrapeInterval: 15s
```

Comment:
just remove the whole block so we use the defaults?

Reply:
The default is 1m, doesn't look right for our tests: https://prometheus.io/docs/prometheus/latest/configuration/configuration/

Reply:
https://github.com/istio/istio/blob/054436c4f5007b055b4382965fcad202431eb197/install/kubernetes/helm/istio/charts/prometheus/values.yaml#L35 is where it is set

Reply:
ahaa I looked at normal istio values.yaml file and it was not there over there..
Thanks, removed the default from values-e2e.yaml